modules/@apostrophecms/submitted-draft/ui/apos/components/AposSubmittedDraftAdminBarIcon.vue

```diff
@@ -0,0 +1,62 @@
+<template>
+  <span v-if="count > 0" class="apos-submitted-draft-admin-bar-container">
+    <AposIndicator
+      icon="tray-full-icon"
+      class="apos-input-icon apos-submitted-draft-admin-bar-icon"
+      :icon-size="size"
+    />
+    <span v-if="canPublish" class="apos-submitted-draft-admin-bar-counter">
+      {{ count }}
+    </span>
+  </span>
+</template>
+<script>
+export default {
+  name: 'AposSubmittedDraftAdminBarIcon',
+  props: {
+    size: {
+      type: Number,
+      required: true,
+      default() {
+        return null;
+      }
+    }
+  },
+  data() {
+    return {
+      count: 0
+    };
+  },
+  computed: {
+    canPublish() {
+      return window.apos.modules['@apostrophecms/submitted-draft'].canPublish;
+    }
+  },
+  mounted() {
+    this.updateCount();
+  },
+  destroyed() {
+    if (this.timeout) {
+      clearTimeout(this.timeout);
+    }
+  },
+  methods: {
+    async updateCount() {
+      this.count = (await apos.http.get(apos.modules['@apostrophecms/submitted-draft'].action, {
+        qs: {
+          count: 1,
+          'apos-mode': 'draft'
+        }
+      })).count;
+      // Not declared in data() because it is not reactive
+      this.timeout = setTimeout(this.updateCount, 10000);
+    }
+  }
+};
+</script>
+<style lang="scss" scoped>
+  .apos-submitted-draft-admin-bar-container {
+    /* TODO: hack to make it visible for now */
```

Comment:
This shouldn't be necessary. Remove `apos-input-icon` above (I'm not sure what it's doing there) and the icon is no longer positioned absolute. Then you don't need to set the width. You'll only need to swap the order of the icon and counter, maybe.